VirtoCommerce.Platform.Core/Notifications/SearchNotificationCriteria.cs

```diff
@@ -1,15 +1,12 @@
+using System;
 using System.Linq;
 using VirtoCommerce.Platform.Core.Common;
 
 namespace VirtoCommerce.Platform.Core.Notifications
 {
 	public class SearchNotificationCriteria : ValueObject
     {
-        public SearchNotificationCriteria()
-        {
-            RepeatHoursIntervalForFail = 5;
-        }
-		public int Take { get; set; }
+        public int Take { get; set; }
```

Comment:
you forgot to add  the default value for  RepeatMinutesIntervalForFail property (previous value was 5 hours)